lib/topological_inventory/providers/common/operations/source.rb

```diff
@@ -121,7 +124,11 @@ def endpoint
         end
 
         def authentication
-          @authentication ||= api_client.fetch_authentication(source_id, endpoint)
+          @authentication ||= if endpoint.receptor_node.present?
+                                "n/a"
```

Comment:
I suppose it'll be tested somewhere against this value so I would prefer constant here

Reply:
done!